packages/manager/modules/exchange/src/external-contact/external-contact.html

```diff
@@ -3,220 +3,117 @@
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_creationDate"
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_hiddenFromGAL"
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_state"
-                            ></th>
-                            <th class="min-width text-center"></th>
-                        </tr>
-                    </thead>
-
-                    <tbody data-ng-if="ctrl.contactsLoading">
-                        <tr>
-                            <td colspan="6" class="text-center">
-                                <oui-spinner></oui-spinner>
-                            </td>
-                        </tr>
-                    </tbody>
-
-                    <tbody
-                        data-ng-if="!ctrl.contactsLoading && ctrl.contacts.list.results.length === 0"
+                            data-translate="exchange_tab_EXTERNAL_CONTACTS_table_actions_menu_modify"
+                        ></span>
+                    </oui-action-menu-item>
+                    <oui-action-menu-item
+                        on-click="setAction('exchange/external-contact/remove/external-contact-remove', $row.externalEmailAddress)"
+                        data-disabled="!ctrl.services.exchangeStates.constructor.isOk($row)"
                     >
-                        <tr>
-                            <td
-                                class="text-center"
-                                colspan="6"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_empty"
-                            ></td>
-                        </tr>
-                    </tbody>
+                        <span
+                            data-translate="exchange_tab_EXTERNAL_CONTACTS_table_actions_menu_delete"
+                        ></span>
+                    </oui-action-menu-item>
+                </oui-action-menu>
 
-                    <tbody data-ng-if="!ctrl.contactsLoading">
-                        <tr
-                            data-ng-repeat="element in ctrl.contacts.list.results track by $index"
+                <extra-top>
+                    <oui-action-menu text="Actions" align="start">
+                        <oui-action-menu-item
+                            text="{{'exchange_tab_EXTERNAL_CONTACTS_configuration_contact_add_title_button' | translate}}"
```

Comment:
Could you please avoid using the deprecated `text` attribute?

Prefer using a transclude.

```html
<span data-translate="exchange_tab_EXTERNAL_CONTACTS_configuration_contact_add_title_button"></span>
```

See: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/action-menu#deprecated